src/animatediff/generate.py

```diff
@@ -1148,9 +1146,6 @@ def preview_callback(i: int, video: torch.Tensor, save_fn: Callable[[torch.Tenso
 
     seed_everything(seed)
 
-    logger.info(f"{len( region_condi_list )=}")
-    logger.info(f"{len( region_list )=}")
-
     pipeline_output = pipeline(
```

Comment:
Basically, don't delete the logs left in the code.
Most of the time, they are still being debugged or are left to investigate behavior that could be improved.